revdep/problems.md

```diff
@@ -90,6 +90,82 @@ See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
 manual.
 ```
 
+## RJDBC (0.2-5)
+Maintainer: Simon Urbanek <Simon.Urbanek@r-project.org>
+
+0 errors | 1 warning  | 1 note 
+
+```
+checking for missing documentation entries ... WARNING
+Undocumented S4 methods:
+  generic 'dbDataType' and siglist 'JDBCConnection,ANY'
+All user-level objects in a package (including S4 classes and methods)
+should have documentation entries.
+See chapter ‘Writing R documentation files’ in the ‘Writing R
+Extensions’ manual.
```

Comment:
@hadley: These errors are new, I think caused by DBI now properly exporting dbDataType().

In RJDBC, the dbDataType() function is [documented](https://github.com/cran/RJDBC/blob/8005611399db6882628bc6a9590921658f6248da/man/JDBCConnection-methods.Rd#L53-L54) and [implemented](https://github.com/cran/RJDBC/blob/8005611399db6882628bc6a9590921658f6248da/R/class.R#L219-L224) with the (Connection, ANY) signature, in DBI it's only (DBIObject). Nothing seems to change if I use the (DBIObject, ANY) signature.

I guess there isn't anything we can/should do about this in DBI?